Square.cpp

```diff
@@ -1,15 +1,30 @@
 #ifndef Square_cpp
 #define Square_cpp
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
 using namespace std;
 
-class Square : public Draw
+class Square : public Shape
 {
 public:
     void drawShape() override
     {
         cout << "square" << endl;
     }
+
+    int inputAndCalculateArea() override
+    {
+        int length, width, area;
+        cout <<"Enter square dimensions" <<endl;
+        cout <<"Enter Length" <<endl;
+        cin >> length ;
+        cout <<"Enter Width" <<endl;
+        cin >> width ;
+        area = length*width;
+        cout << "Area = "  << area;
+        return 0;
```

Comment:
Why are we returning `0`? Isn't the function `inputAndCalculateArea` suppose to return area? Will it always remain `0` no matter user's input?